mlflow/utils/databricks_utils.py

```diff
@@ -143,7 +144,7 @@ def _fail_malformed_databricks_auth(profile):
                           "https://github.com/databricks/databricks-cli." % profile)
 
 
-def get_databricks_host_creds(profile=None, path=None):
+def get_databricks_host_creds(server_uri=None):
     """
     Reads in configuration necessary to make HTTP requests to a Databricks server. This
     uses the Databricks CLI's ConfigProvider interface to load the DatabricksConfig object.
```

Comment:
Let's add that if no Databricks CLI profile matching the ``server_uri`` is available, we will also try to retrieve it from the Databricks secret manager if available, and in that case ``server_uri`` needs to have the path part which is the key prefix, etc. we should document how to use the secret manager for mlflow in databricks docs, but within the MLflow repo this is likely the only place that will explain it so let's make it more comprehensive.

Reply:
Got it - will add documentation around this.